src/mappings/dayUpdates.ts

```diff
@@ -130,3 +130,24 @@ export function updateTokenDayData(token: Token, event: ethereum.Event): TokenDa
 
   return tokenDayData as TokenDayData
 }
+
+export function updateUniqueDailyInteractions(event: ethereum.Event): DailyUniqueInteraction {
+  let timestamp = event.block.timestamp.toI32()
+  let dayID = timestamp / 86400
+  let dayStartTimestamp = dayID * 86400
```

Comment:
these can be const.
And the last one is the floor of the timestamp. Isn't there a better way to compute it?

Reply:
I agree with Jorge about `dayID`. This can be`YYYY-MM-DD` format. There's no native `Date` type in the graph implementation of AssemblyScript AFAIK. Happy to help

Reply:
By doing something like this (idk if it's the best approach):

```ts
  const timestamp = event.block.timestamp.times(BigInt.fromString('1000')).toI64()
  const today = new Date(timestamp)

  const paddedMonth = (today.getUTCMonth() + 1).toString().padStart(2, '0')
  const paddedDay = today
    .getUTCDate()
    .toString()
    .padStart(2, '0')

  const dayIdString = `${today.getUTCFullYear()}-${paddedMonth}-${paddedDay}`

  let uniqueDailyInteractionsData = DailyUniqueInteraction.load(dayIdString)

  if (uniqueDailyInteractionsData === null) {
    uniqueDailyInteractionsData = new DailyUniqueInteraction(dayIdString)
    uniqueDailyInteractionsData.date = timestamp

    uniqueDailyInteractionsData.addresses = []
  }

  uniqueDailyInteractionsData.save()
```

The entities would be:

```json
{
  "id": "2021-09-03",
  "date": "1630703384000",
  "addresses": [
    "0x35e2acd3f46b13151bc941daa44785a38f3bd97a",
    "0xf90adbce4ab7b51756a32228edddd17a4b9af2a2",
    "0xf856099cdf925b60b4c2cb8afe9040206beeaa61"
  ]
}
```